### PR TITLE
VA Template - Updated to 4.5.1 and new streaming extensions

### LINKED ISSUE #1852: Update streaming extensions package
- [ ] VA template & sample
- [ ] Skill template & sample

*dependent on #1854

### PR DESCRIPTION
**Pull request by lauren-mills**
Closes #1852 
Adds to #1854 

### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
- Lib updates in PR
- Skill template updates coming
- Skills projects coming

#### Deployment Scripts
- [x] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects

### CODE REVIEW

File: templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj [hunk 1]
```diff
@@ -17,21 +14,22 @@
...
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.StreamingExtensions" Version="4.5.1-preview1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Skills" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Solutions" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.TemplateManager" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Schema" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.StreamingExtensions" Version="4.5.1-preview1" />
```
Comment (lzc850612) [inline templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj]: They have two assemblies 
Microsoft.Bot.Builder.StreamingExtensions and Microsoft.Bot.StreamingExtensions?

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj (hunk 1 above)
Comment (lauren-mills) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj]: Yes, there are two. https://www.nuget.org/packages/Microsoft.Bot.StreamingExtensions/4.5.0-preview1 and https://www.nuget.org/packages/Microsoft.Bot.Builder.StreamingExtensions/4.5.0-preview1 but I need to check if I can remove the first one in our project. I'm not sure what's different between them

Comment (lauren-mills) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj]: The project still builds without the Microsoft.Bot.StreamingExtensions package so I guess its safe to remove?

Comment (lzc850612) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/VirtualAssistantSample.csproj]: yes i think we should remove Microsoft.Bot.StreamingExtensions.

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**